lib/Swagger/src/Discussion/Api/ContributionApi.php

```diff
@@ -0,0 +1,348 @@
+<?php
```

Comment:
I'm wondering what's the policy for those clients. Can we generate the client only for one resource or simply remove the generated classes for unnecessary endpoints?